test/e2e_node/e2e_node_suite_test.go

```diff
@@ -89,7 +89,7 @@ func TestE2eNode(t *testing.T) {
 			glog.Errorf("Failed creating report directory: %v", err)
 		} else {
 			// Configure a junit reporter to write to the directory
-			junitFile := fmt.Sprintf("junit_%02d.xml", *junitFileNumber)
+			junitFile := fmt.Sprintf("junit_%s%02d.xml", framework.TestContext.ReportPrefix, config.GinkgoConfig.ParallelNode)
```

Comment:
Yeah, ginkgo framework will make sure that. We also use the same trick in e2e framework. https://github.com/kubernetes/kubernetes/blob/master/test/e2e/e2e.go#L235